src/libraries/System.Private.CoreLib/src/System/Globalization/IcuLocaleData.cs

```diff
@@ -686,3100 +605,3131 @@ static void GenerateData (string[] cultures, (string, string)[] lcids)
-            0x21 << 16 | 1651 << 4 | 2,      // id
-            0x22 << 16 | 3069 << 4 | 2,      // uk
-            0x23 << 16 | 231 << 4 | 2,       // be
-            0x24 << 16 | 2647 << 4 | 2,      // sl
-            0x25 << 16 | 1182 << 4 | 2,      // et
-            0x26 << 16 | 2005 << 4 | 2,      // lv
-            0x27 << 16 | 1983 << 4 | 2,      // lt
-            0x28 << 16 | 2943 << 4 | 2,      // tg
-            0x29 << 16 | 1198 << 4 | 2,      // fa
-            0x2a << 16 | 3141 << 4 | 2,      // vi
-            0x2b << 16 | 1629 << 4 | 2,      // hy
-            0x2c << 16 | 200 << 4 | 2,       // az
-            0x2d << 16 | 1187 << 4 | 2,      // eu
-            0x2e << 16 | 1611 << 4 | 3,      // hsb
-            0x2f << 16 | 2056 << 4 | 2,      // mk
-            0x30 << 16 | 2849 << 4 | 2,      // st
-            0x31 << 16 | 3004 << 4 | 2,      // ts
-            0x32 << 16 | 2979 << 4 | 2,      // tn
-            0x33 << 16 | 3136 << 4 | 2,      // ve
-            0x34 << 16 | 3186 << 4 | 2,      // xh
-            0x35 << 16 | 3383 << 4 | 2,      // zu
-            0x36 << 16 | 15 << 4 | 2,        // af
-            0x37 << 16 | 1750 << 4 | 2,      // ka
-            0x38 << 16 | 1244 << 4 | 2,      // fo
-            0x39 << 16 | 1596 << 4 | 2,      // hi
-            0x3a << 16 | 2130 << 4 | 2,      // mt
-            0x3b << 16 | 2583 << 4 | 2,      // se
-            0x3c << 16 | 1501 << 4 | 2,      // ga
-            0x3d << 16 | 3203 << 4 | 2,      // yi
-            0x3e << 16 | 2115 << 4 | 2,      // ms
-            0x3f << 16 | 1797 << 4 | 2,      // kk
-            0x40 << 16 | 1913 << 4 | 2,      // ky
-            0x41 << 16 | 2874 << 4 | 2,      // sw
-            0x42 << 16 | 2974 << 4 | 2,      // tk
-            0x43 << 16 | 3084 << 4 | 2,      // uz
-            0x44 << 16 | 3009 << 4 | 2,      // tt
-            0x45 << 16 | 269 << 4 | 2,       // bn
-            0x46 << 16 | 2308 << 4 | 2,      // pa
-            0x47 << 16 | 1539 << 4 | 2,      // gu
-            0x48 << 16 | 2293 << 4 | 2,      // or
-            0x49 << 16 | 2906 << 4 | 2,      // ta
-            0x4a << 16 | 2926 << 4 | 2,      // te
-            0x4b << 16 | 1824 << 4 | 2,      // kn
-            0x4c << 16 | 2061 << 4 | 2,      // ml
-            0x4d << 16 | 183 << 4 | 2,       // as
-            0x4e << 16 | 2110 << 4 | 2,      // mr
-            0x4f << 16 | 2540 << 4 | 2,      // sa
-            0x50 << 16 | 2066 << 4 | 2,      // mn
-            0x51 << 16 | 279 << 4 | 2,       // bo
-            0x52 << 16 | 392 << 4 | 2,       // cy
-            0x53 << 16 | 1819 << 4 | 2,      // km
-            0x54 << 16 | 1966 << 4 | 2,      // lo
-            0x55 << 16 | 2141 << 4 | 2,      // my
-            0x56 << 16 | 1511 << 4 | 2,      // gl
-            0x57 << 16 | 1839 << 4 | 3,      // kok
-            0x58 << 16 | 2098 << 4 | 3,      // mni
-            0x59 << 16 | 2563 << 4 | 2,      // sd
-            0x5a << 16 | 2
```

Comment:
Change to deal with the sort order prefix; they are grouped into sections (same order as before); however the sort is used for the lookup e.g. LCID lookup of `de-de` vs `de-de_phoneb`

Reply:
`LCIDToLocaleName` method; GitHub doesn't seem to be highlighting the correct lines 😢